src/components/dialog/dialog.js

```diff
@@ -521,7 +522,8 @@ function MdDialogProvider($$interimElementProvider) {
         '      <p>{{::dialog.mdTextContent}}</p>',
         '    </div>',
         '    <md-input-container md-no-float ng-if="::dialog.$type == \'prompt\'" class="md-prompt-input-container">',
```

Comment:
@DevVersion should this be changed to `===` instead of `==` while you're in the area?

PR looks good, thanks for doing this.